app/models/concerns/starter_code_importable.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module StarterCodeImportable
+  extend ActiveSupport::Concern
+
+  def starter_code?
+    starter_code_repo_id.present?
+  end
+
+  def starter_code_repository
+    return unless starter_code?
+    @starter_code_repository ||= GitHubRepository.new(creator.github_client, starter_code_repo_id)
+  end
+
+  def template_repos_enabled?
+    template_repos_enabled
+  end
+
+  def template_repos_disabled?
```

Comment:
We typically don't define methods like this in Ruby. Prefer `!template_repos_enabled?`